CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 ### Added
 ### Fixed
+* Fix class properties should have defaults as standard js classes would have (#2390)
```

Comment:
You don't need to document this since it hasn't been released yet. (I guess technically it has as an alpha version, but we don't have an entry in this for that, so...).

Reply:
So remove this? I added because of the PR template "Have you updated CHANGELOG.md?"

Reply:
Looks good where you moved it to now. 👍

Reply:
I meant delete since it cancels itself out (broken in 3.0.0 and fixed in... 3.0.0) but nbd